Protocol/Error Messages/Protocol/QActions/QAction/CSharpCoreInterAppBrokerSupport.cs

```diff
@@ -26,7 +26,7 @@ public static IValidationResult InvalidInterAppReplyLogic(IValidate test, IReada
                 Source = Source.Validator,
                 FixImpact = FixImpact.NonBreaking,
                 GroupDescription = "",
-                Description = String.Format("Invocation of method '{0}.{1}' is not compatible with '{2}'. QAction ID '{3}'.", methodClassType, incompatibleMethod, "DataMiner.Core.InterApp > v1.0.1.0", qactionId),
+                Description = String.Format("Invocation of method '{0}.{1}' is not compatible with '{2}'. QAction ID '{3}'.", methodClassType, incompatibleMethod, "DataMiner.Core.InterApp > v1.0.1.1", qactionId),
```

Comment:
1.0.1.0 was correct. The first version released is 1.0.1.1  and there was a maximum length in the windows form app so I couldn't write >=1.0.1.1 hence the >1.0.1.0

Reply:
I'll have a look at the max length.